src/board.cpp

```diff
@@ -0,0 +1,223 @@
+#include "board.h"
+#include "block.h"
+
+constexpr int movesROW[4] = {0, +1, 0, -1};
+constexpr int movesCOL[4] = {-1, 0, +1, 0};
+
+Board::Board(unsigned int _HIGHT = 20, unsigned int _WIDTH = 10): 
+    HIGHT(_HIGHT), WIDTH(_WIDTH), 
+    board(std::vector<std::vector<bool>>(HIGHT, std::vector<bool>(WIDTH, false))), 
+    blocktype(std::vector<std::vector<BlockType>>(HIGHT, std::vector<BlockType>(WIDTH, (BlockType) 0))),
```

Comment:
To jest okropne, ale na razie nie da się lepiej, napiszę potem jakąś klasę tablicy 2D